e2e_samples/fabric_dataops_sample/docs/adr_naming_convention.md

```diff
@@ -0,0 +1,79 @@
+# Naming convention for Fabric E2E sample
+
+This document describes the naming convention for the Fabric E2E sample. It includes the naming convention for the following:
+
+- Git Folders
+- Scripts (python, shell, powershell etc.)
+- Configuration files (.yml, .json, .cfg etc.)
+- Azure/Fabric resources
+
+## Status
+
+Proposed
+
+## Context
+
+As the Fabric E2E sample is a complex solution with multiple components, it is important to have a consistent naming convention for the various components. This will help in better organization and management of the solution. It would also help multiple developers to work on the solution without any confusion.
```

Comment:
This is not just for the Fabric sample - the idea is to be cross-repo.
```suggestion
As the DataOps repository contains several and complex solutions with multiple components, it is important to have a consistent naming convention for the individual and collective elements that compose the repo. This norm will help to  better organize and manage of the solution and additionally will allow multiple developers to work effectively without any confusion.
```

Reply:
This ADR is specifically for the Fabric E2E sample.

Reply:
Lets aim to close this PR initially targetting just the Fabric E2E sample, but open a tasks to create a repo-wide naming convention standard as it may require additional rework to the existing single tech sample in Fabric.